### ISSUE TITLE
Runner#stop raises an exception

### ISSUE THREAD

**Issue by kleinron** [jondot/sneakers#407]
Runner#stop calls server-engine's stop method. The latter expects `graceful` argument, which is not provided.

As a temp workaround I redefined as follows:

```ruby
module Sneakers
  class Runner
    def stop(graceful=true)
      @se.stop(graceful)
    end
  end
end
```

**Comment by gabrieljoelc** [comment]
@kleinron thanks for the suggestion. wanna open a PR? 🙏

**Comment by gabrieljoelc** [comment]
Closed via #409
